src/Lykke.Service.PayInvoicePortal/Controllers/User/ResetPasswordController.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Net;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Common;
+using Common.Log;
+using Lykke.Common.Api.Contract.Responses;
+using Lykke.Common.Log;
+using Lykke.Service.PayAuth.Client;
+using Lykke.Service.PayAuth.Client.Models;
+using Lykke.Service.PayAuth.Client.Models.Employees;
+using Lykke.Service.PayInvoice.Client;
+using Lykke.Service.PayInvoicePortal.Models.User;
+using LykkePay.Common.Validation;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Mvc;
+using ErrorResponseException = Lykke.Service.PayAuth.Client.ErrorResponseException;
+
+namespace Lykke.Service.PayInvoicePortal.Controllers.User
+{
+    public class ResetPasswordController : Controller
+    {
+        private const string ViewPath = "../User/ResetPassword";
+        private const int TokenHexLength = 40;
+
+        private readonly IPayAuthClient _payAuthClient;
+        private readonly IPayInvoiceClient _payInvoiceClient;
+        private readonly ILog _log;
+        private readonly Regex _tokenRegex = new Regex(@"[\dA-F]+");
+
+        public ResetPasswordController(
+            IPayAuthClient payAuthClient,
+            IPayInvoiceClient payInvoiceClient,
+            ILogFactory logFactory)
+        {
+            _payAuthClient = payAuthClient;
+            _payInvoiceClient = payInvoiceClient;
+            _log = logFactory.CreateLog(this);
+        }
+
+        [HttpGet]
+        [Route("resetPassword")]
```

Comment:
api/resetPassword?

Reply:
this method is for MVC view, another one is for API call, that's why such route.